canonical-websites/www.ubuntu.com
# Embedded dynamic form

## Done
Move the current embedded form into the correct structure.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /core and /internet-of-things
- Check the contact buttons/links open the interactive form
- Submit the form and check it goes through

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4593
